### ISSUE TITLE
FPS-related issues

### ISSUE THREAD

**Issue by txw-github** [lyuwenyu/RT-DETR#297]
Hello, may I ask why I am using pytorch version on RTX3060, the training dataset with 6000 images takes 4-5min, and the test set with 2000 images takes 45s. whereas yolov8 only takes 1.3min and 17s, please guide me, I need it urgently.
